source/compiler_identification.d

```diff
@@ -30,6 +30,9 @@ struct Compiler
 
     ///Accepts both a complete path to an executable or a global environment search path name
     string binOrPath;
+
+    ///Librarian tool
+    string archiver = "llvm-ar";
```

Comment:
This may be added as a linker sent. I still need to think a little more about it